packages/shorebird_cli/integration_test/shorebird_cli_integration_test.dart

```diff
@@ -1,19 +1,104 @@
 import 'dart:io';
 
+import 'package:mason_logger/mason_logger.dart';
+import 'package:path/path.dart' as p;
 import 'package:test/test.dart';
 
 void main() {
-  test('--version', () async {
-    final result = await Process.run(
+  final logger = Logger();
+
+  ProcessResult runShorebirdCommand(
```

Comment:
minor but any reason not to make this even more generic (e.g. `run` instead of `runShorebirdCommand`)?

Reply:
Good call, updated.